### PR TITLE
[DGPFUS-37] Database Design For A Transportation Company

### PR DESCRIPTION
**Pull request by bukiz-cloud**
A created a DATABASE for a Transportation company

### CODE REVIEW

File: submissions/bukunmiAdeniyi/database1/init.sql [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+     FOREIGN KEY (passengers_id) REFERENCES passengers (id)
+      );
+      SHOW COLUMNS FROM passengers_details;
+      
+      -- CREATING a TABLE indicating events of accidents whether the passengers survived or not -- 
+      CREATE TABLE accident_trips (
+    id INT NOT NULL AUTO_INCREMENT ,
+    passenger_id INT NOT NULL,
+    accident_id INT NOT NULL,
+    passenger_status ENUM('Survived', 'Not Survived', 'comtemplating'),
```
Comment (ToluEnny24) [inline submissions/bukunmiAdeniyi/database1/init.sql:43]: contemplating kwa? Boolean is fine, yes or no is fine

File: submissions/bukunmiAdeniyi/database1/init.sql [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+created_at DATETIME,
+PRIMARY KEY (id)
+);
+ALTER TABLE passengers DROP passenger_id;
+SHOW COLUMNS FROM passengers;
+
+-- CREATING a TABLE keeping the records of passengers 
+ CREATE TABLE passengers_details (
+     id INT NOT NULL AUTO_INCREMENT,  
+     passenger_class VARCHAR(100) NOT NULL,
```
Comment (ToluEnny24) [inline submissions/bukunmiAdeniyi/database1/init.sql:24]: passenger_class should be ENUM class 1, 2 or 3

**Review by ToluEnny24 — CHANGES_REQUESTED** [review]
Please look through and effect changes immediately

**Review by bukiz-cloud — COMMENTED** [review]
The necessary changes have been done

**Review by ToluEnny24 — APPROVED** [review]

**Pull request merged**